OpenRA.Mods.Common/Traits/Render/WithVoxelBody.cs

```diff
@@ -39,7 +39,7 @@ public class WithVoxelBodyInfo : ConditionalTraitInfo, IRenderActorPreviewVoxels
 		}
 	}
 
-	public class WithVoxelBody : ConditionalTrait<WithVoxelBodyInfo>, IAutoSelectionSize
+	public class WithVoxelBody : ConditionalTrait<WithVoxelBodyInfo>, IAutoSelectionSize, IAutoRenderSize
```

Comment:
You create `AutoRenderSize` as a separate trait for sprite actors, but implement the interface directly here.
For consistency we should go with either a second trait that works with `WithVoxelBody` the way `AutoRenderSize` works with `RenderSprites`, or get rid of `AutoRenderSprites` and implement the interface directly.

Reply:
Implementing the interface directly on WithSpriteBody or RenderSprites would be too problematic (since there must be only 1 trait with that interface per actor) and was dropped for that reason (this was also the main motivation when making AutoSelectionSize and CustomSelectionSize separate traits).

I see your point about consistency, but if possible I don't want to do the split for voxels in this PR yet, since that would make the PR and upgrade rules grow further without any practical benefit. We only really need the `Auto-/CustomRenderSize` traits for sprites, since there are several special cases that would otherwise be hard to get right, while I don't know of any voxels that would need this.

This is all about laying the groundwork to fix issues like bibs plopping in and out of the screen because they don't count towards the render bounds.